mesa_frames/abstract/agentsetregistry.py

```
@@ -74,47 +74,88 @@ def __init__(self) -> None: ...
             The updated AbstractAgentSetRegistry.
         """
         with suppress(KeyError, ValueError):
-            return self.remove(agents, inplace=inplace)
+            return self.remove(sets, inplace=inplace)
         return self._get_obj(inplace)
 
+    @abstractmethod
+    def rename(
+        self,
+        target: (
+            mesa_frames.abstract.agentset.AbstractAgentSet
```

Ruff (0.12.2)

Undefined name `mesa_frames`

(F821)